table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java

```diff
@@ -146,70 +155,142 @@ public String getName() {
     }
   }
 
-  private PathTranslator mountAlluxioPaths(Table table, List<Partition> partitions,
-      UdbBypassSpec bypassSpec)
-      throws IOException {
-    String tableName = table.getTableName();
-    AlluxioURI ufsUri;
-    AlluxioURI alluxioUri = mUdbContext.getTableLocation(tableName);
-    String hiveUfsUri = table.getSd().getLocation();
+  /**
+   * Mounts the database to Alluxio filesystem. This method mounts tables and partitions
+   * (if applicable) within this database to Alluxio filesystem.
+   *
+   * @param bypassSpec bypass spec
+   */
+  private void mount(UdbBypassSpec bypassSpec) throws IOException {
+    List<String> tableNames = getTableNames();
+    Map<Table, List<Partition>> tables = new HashMap<>(tableNames.size());
+    for (String tableName : tableNames) {
+      Table table;
```

Comment:
If we only use the `table` in the try block, just define it in the try block

Reply:
Fixed in 047139a5b49da41a4c8da063897dd0f39fee5098.